backend/backend/backend-src/lib/events.ts

```diff
@@ -2,10 +2,45 @@
  * @file Handlers related to events and balances.
  */
 
-import {Request, Response} from 'express'
+import {Request, Response, Router} from 'express'
 import * as Checks from './checks'
 import * as dbEvents from './db/events'
 
+export function createEventsRouter(): Router {
+  const router = Router()
+  router.get('/:address/:ticker', async (req, res, next) => {
+    try {
+      await handleGetPackageEvents(req, res)
+    } catch (error) {
+      next(error)
+    }
+  })
+  router.get('/:address/:ticker', handleGetPackageEvents)
```

Comment:
Shouldn't this line also be inside a try/catch block?

Reply:
Done

Reply:
That PR created from this branch. thats why this code is already there.